index.js

```diff
@@ -222,6 +224,12 @@ class ResembleHelper extends Helper {
         return boundingBox;
     }
 
+    {
+   const allureReporter = Container.plugins('allure');
+        if (allureReporter) {
+          allureReporter.addAttachment('Screenshot diffImage', fs.readFileSync(path.join(global.output_dir, fileName)), 'image/png');
```

Comment:
Use the actual Diff Image name, using Test Names

Reply:
Tried.